extensions/Needinfo/template/en/default/hook/request/email-after_summary.txt.tmpl

```diff
@@ -8,6 +8,27 @@
 
 [% RETURN UNLESS flag && flag.type.name == 'needinfo' && flag.status == '?' %]
 ---
---- This request has set a needinfo flag on the [% terms.bug %].
---- You can clear it by logging in and replying in a comment.
+--- Hello from [% terms.Bugzilla %], the [% terms.bug %] tracker for Mozilla and Firefox:
+---
+--- Someone has asked for you for information about this [% terms.bug %].
+---
+--- Please go to [%+ urlbase %]show_bug.cgi?id=[% bug.bug_id %] to see the
+--- question and respond to it.
```

Comment:
Stating that they need to visible the bug to see the question might be confusing as the question is part of this email; the contents of this hook is inserted at the top of the email template rather than replacing it.

Reply:
Let me dig back through the request email template. I was hoping not to have to touch it.

Reply:
You shouldn't have to touch the base template at all.  The wording here could be adjusted to something like:
```
--- Someone has asked you for information about his bug.
---
--- Visit https://bugzilla.mozilla.org/123456 to respond to the question below.
```